CHANGELOG.md

```diff
@@ -59,6 +59,7 @@
 * Upgraded Go code dependencies ([#2828], [#2826], [#2813])
 * Rename `ncurses` to `libncurses` ([#2769])
 * Update schnauzer's registry map ([#2867])
+* Fix AWS Rust SDK API calls and surface AWS Rust SDK errors in pubsys ([#2895])
```

Comment:
It's not clear to me that this is required, as someone going from the previous version of Bottlerocket to the latest would not notice this, but I don't think it hurts.

Reply:
Very true. This was only an intermitent problem that someone would have seen if they were building / publishing builds off our `develop` branch after we bumped AWS Rust SDK. So nothing really changed for users going between releases.